pkgs/by-name/wf/wfview/package.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  eigen,
+  hidapi,
+  libopus,
+  libpulseaudio,
+  portaudio,
+  qt6,
+  qt6Packages,
+  rtaudio,
+}:
+
+stdenv.mkDerivation (finalAttr: {
```

It doesn't matter but

```suggestion
stdenv.mkDerivation (finalAttrs: {
```